source/fuzz/fuzzer_util.h

```diff
@@ -497,6 +498,16 @@ opt::Instruction* GetLastInsertBeforeInstruction(opt::IRContext* ir_context,
                                                  uint32_t block_id,
                                                  SpvOp opcode);
 
+// Returns true if all of the members of OpTypeStruct instruction with result
+// id |struct_type_id| are decorated with BuiltIn decoration.
+// Otherwise (none of the members are BuiltIn), returns false.
+//
+// According to the spec for the BuiltIn decoration:
+// - When applied to a structure-type member, all members of that structure
+//   type must also be decorated with BuiltIn. (No allowed mixing of built-in
+//   variables and non-built-in variables within a single structure.)
+bool HasBuiltInMembers(opt::IRContext* ir_context, uint32_t struct_type_id);
```

Comment:
Can you please:

- change the name to "MembersHaveBuiltInDecoration"

- change the comment to (something like): "Requires that |struct_type_id| is the id of a struct type, and (as per the SPIR-V spec) that either all or none of the members of |struct_type_id| have the BuiltIn decoration.  Returns true if and only if all members have the BuiltIn decoration."

Reply:
Fixed in 9868ff3.